lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -63,7 +63,11 @@ def purchase(money, payment_method, options = {})
       def authorize(money, payment_method, options = {})
         requires!(options, :order_id)
         payment_details = payment_details(payment_method, options)
-        authorize_request(money, payment_method, payment_details.merge(options))
+        if options[:account_funding_transaction]
+          aft_request(money, payment_method, payment_details.merge(**options))
+        else
+          authorize_request(money, payment_method, payment_details.merge(options))
+        end
       end
```

Comment:
As discussed in the ticket, this change was additive only. I left the AFT logic in the `credit` method until we determine it should be removed/updated according to Worldpay's docs